app/jobs/views.py

```diff
@@ -0,0 +1,33 @@
+# Third-Party imports
+from django.contrib import messages
+from django.shortcuts import render
+from django.template.response import TemplateResponse
+
+# gitcoin-web imports
+from .forms import JobForm
+
+
+def list_jobs(request):
+    context = dict()
```

Comment:
Drop this and simply `context={}` if that's what you're going for here.  Realistically, we probably want to pass some contextual data to the list view.